### DISCUSSION TITLE
Menu组件 horizontal模式 箭头问题

### DISCUSSION THREAD

**Discussion by consistent-k** [ant-design/ant-design#43672] [Q&A] [answered]
![image](https://github.com/ant-design/ant-design/assets/67736912/91e61ab3-c21b-4287-81bd-60a83b9ac78a)
如图所示，此处菜单有子菜单，但是如果不hover无法感知，是否可以增加配置项，这里可以传入一个icon，比如下拉箭头。

**Comment by linxianxi** [comment, marked as answer]
可以先这样
`label: <>Navigation Three - Submenu <DownOutlined /></>`

**Reply by consistent-k** [reply]
![image](https://github.com/ant-design/ant-design/assets/67736912/d81b86f7-ab6e-4fe9-b703-4f99d1983d8e)
 下层还有subMenu的时候会这样，哈哈哈  不过按照你得思路 我只在第一层菜单加这个应该就好了。
